test/http/basic_parser_v1.cpp

```diff
@@ -236,13 +189,22 @@ class basic_parser_v1_test : public beast::detail::unit_test::suite
     parse(boost::string_ref const& m, F&& f)
     {
         using boost::asio::buffer;
+        for(;;)
```

Comment:
`do {...} while(false);` is clearer to me. We could remove one `break` and it makes it clearer that it's not really a loop.